its/projects/WebApplication/WebApplication.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Do we really need a project with this complicated structure? I'd prefer to have something simple that will just demonstrate what we need. This one is large and hard to get oriented in.

Reply:
I suggest to create an empty project with a html file or two. `Foo.vb` and `Program.cs` or other necessary minimum. There are really many things going on that we don't care about.